pkg/arm/v5/resources.go

```diff
@@ -215,12 +241,98 @@ func (g *simpleGenerator) lbKubernetes() *network.LoadBalancer {
+				},
 			},
 			BackendAddressPools: &[]network.BackendAddressPool{
 				{
 					Name: to.StringPtr(lbKubernetesBackendPoolName),
 				},
 			},
+			LoadBalancingRules: &[]network.LoadBalancingRule{
+				{
+					LoadBalancingRulePropertiesFormat: &network.LoadBalancingRulePropertiesFormat{
+						FrontendIPConfiguration: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/frontendIPConfigurations/" + lbKubernetesInboundFrontendConfigurationName),
+						},
+						BackendAddressPool: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/backendAddressPools/" + lbKubernetesBackendPoolName),
+						},
+						Probe: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/probes/" + lbKubernetesProbeName443),
+						},
+						Protocol:             network.TransportProtocolTCP,
+						LoadDistribution:     network.SourceIP,
+						FrontendPort:         to.Int32Ptr(443),
+						BackendPort:          to.Int32Ptr(443),
+						IdleTimeoutInMinutes: to.Int32Ptr(15),
+						EnableFloatingIP:     to.BoolPtr(false),
+					},
+					Name: to.StringPtr(lbKubernetesInboundRuleName443),
+				},
+				{
+					LoadBalancingRulePropertiesFormat: &network.LoadBalancingRulePropertiesFormat{
+						FrontendIPConfiguration: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/frontendIPConfigurations/" + lbKubernetesInboundFrontendConfigurationName),
+						},
+						BackendAddressPool: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/backendAddressPools/" + lbKubernetesBackendPoolName),
+						},
+						Probe: &network.SubResource{
+							ID: to.StringPtr(resourceid.ResourceID(
+								g.cs.Properties.AzProfile.SubscriptionID,
+								g.cs.Properties.AzProfile.ResourceGroup,
+								"Microsoft.Network/loadBalancers",
+								lbKubernetesName,
+							) + "/probes/" + lbKubernetesProbeName80),
+						},
+						Protocol:             network.TransportProtocolTCP,
+						LoadDistribution:     network.SourceIP,
```

Comment:
I updated this from `Default` to `SourceIP`.  This generally helps with sessions.  
@jim-minter @mjudeikis  Thoughts?

Reply:
This is good question. It might help with sessions if users are not using VPN or NAT to reach the Loadbalancer. We might build bottleneck without knowing it. But im just thinking outloud. Scrum discussion when @jim-minter is online?